libs/labelbox/src/labelbox/schema/project_model_config.py

```diff
@@ -0,0 +1,34 @@
+from labelbox.orm.db_object import DbObject, Deletable
+from labelbox.orm.model import Field, Relationship
+
+
+class ProjectModelConfig(DbObject):
+    """ A ProjectModelConfig represents an association between a project and a single model config.
+
+    Attributes: 
+        project_id (str): ID of project to associate
+        model_config_id (str): ID of the model configuration
+        model_config (ModelConfig): Configuration for model
+    """
+
+    project_id = Field.String("project_id", "projectId")
```

Comment:
BTW second param is only needed if, camelCased, it differs from the first - do not need to update this PR, FYI